### ISSUE TITLE
Ubuntu 14.04 failure

### ISSUE THREAD

**Issue by sshtmc** [holylobster/nuntius-linux#20]
Hi,

I tried running nuntius on Ubuntu 14.04, and got this error:

> $ nuntius
> obtained bluez profile manager
> 
> *\* (nuntius:10376): WARNING **: application.vala:143: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: Method "RegisterProfile" with signature "osa{sv}" on interface "org.bluez.ProfileManager1" doesn't exist

And then the program just hangs...

**Comment by nacho** [comment]
yeah, I think ubuntu 14.04 is too old for nuntius... would be great if you could check what needs to be changed to support that version of ubuntu too

**Comment by pbor** [comment]
I think that ubuntu still has bluez4 and we use bluez5.

At the very least we should check and exit gracefully with a message instead of hanging.

**Comment by sshtmc** [comment]
Unless you really need features that exist in bluez5 and not in bluez4, it would be much better if you could support both!

**Comment by nacho** [comment]
The problem is that if I recall correct bluez 5 is using dbus and bluez 4 is not....

**Comment by sweeze** [comment]
I've gotten nuntius working on ubuntu vivid with this PPA enabled: https://launchpad.net/~ubuntu-desktop/+archive/ubuntu/transitions/+packages

...fyi if that helps anyone

**Comment by dukex** [comment]
Thanks @sweeze!
